textproc/py-stanfordnlp/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                         1.0
+PortGroup           python                         1.0
+PortGroup           java                           1.0
+PortGroup           github                         1.0
+
+name                py-stanfordnlp
```

Comment:
If this packages is intended to be used as a Python module (with the ```py-*``` name), it should be placed in the "python" category. If, in the other hand, it's a package that happens to be written in Python and is put in a different category it should not start with ```py-```. At least that is my understanding of the consensus.

Reply:
Done—thanks.